st2client/st2client/commands/resource.py

```diff
@@ -731,6 +731,18 @@ def __init__(self, resource, *args, **kwargs):
 
         self.parser.add_argument(argument, metavar=metavar, help=help)
 
+        self.parser.add_argument(
+            "-f",
+            action="store_true",
+            help="Auto yes flag to delete action files from disk.",
+        )
+
+        self.parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Auto yes flag to delete action files from disk.",
+        )
+
```

Comment:
These CLI parser arguments need to be moved to the ActionDeleteCommand. By putting this in the generic ResourceDeleteCommand, these arguments are also showing up for other resources (i.e. `st2 rule delete -h`).  Please review other commands at https://github.com/StackStorm/st2/blob/master/st2client/st2client/commands/action.py for examples to see how to move these arguments to the ActionDeleteCommand.